src/journal.c

```diff
@@ -30,82 +29,82 @@ typedef struct {
+static void JournalEntry_GetLocationEventMisc(u32 locationEvent, JournalEntryLocationEvent *journalEntryLocationEvent);
+static void JournalEntry_GetLocationEventTrainer(u32 locationEvent, JournalEntryLocationEvent *journalEntryLocationEvent);
+static void JournalEntry_GetLocationEventTravel(u32 locationEvent, JournalEntryLocationEvent *journalEntryLocationEvent);
+static void JournalEntry_GetLocationEventItem(u32 locationEvent, JournalEntryLocationEvent *journalEntryLocationEvent);
+static void JournalEntry_GetLocationEventMove(u32 locationEvent, JournalEntryLocationEvent *journalEntryLocationEvent);
+static void JournalEntry_GetOnlineEventBattle(u8 *onlineEvent, JournalEntryOnlineEvent *journalEntryOnlineEvent);
+static void JournalEntry_GetOnlineEventInteraction(u8 *onlineEvent, JournalEntryOnlineEvent *journalEntryOnlineEvent);
+static void JournalEntry_GetOnlineEventTrade(u8 *onlineEvent, JournalEntryOnlineEvent *journalEntryOnlineEvent);
+static void JournalEntry_GetOnlineEventMisc(u8 *onlineEvent, JournalEntryOnlineEvent *journalEntryOnlineEvent);
+static void JournalEntry_GetOnlineEventMinigame(u8 *onlineEvent, JournalEntryOnlineEvent *journalEntryOnlineEvent);
+static u8 JournalEntry_GetGymTooTough(TrainerInfo *trainerInfo, u32 mapID);
 static u8 JournalEntry_TrainerType(u32 trainerID);
 
 static const MapInfo sMapsInfo[] = {
-    { 0x5A, 0x1 },
-    { 0x5B, 0x0 },
-    { 0x5C, 0x1 },
-    { 0x5D, 0x1 },
-    { 0x5E, 0x0 },
-    { 0x5F, 0x1 },
-    { 0x60, 0x1 },
-    { 0x61, 0x1 },
-    { 0x62, 0x1 },
-    { 0x63, 0x0 },
-    { 0x64, 0x1 },
-    { 0x65, 0x0 },
-    { 0x66, 0x1 },
-    { 0x47, 0x0 },
-    { 0x67, 0x0 },
-    { 0x40, 0x0 },
-    { 0x68, 0x0 },
-    { 0x69, 0x1 },
-    { 0x50, 0x1 },
-    { 0x6A, 0x0 },
-    { 0x6B, 0x1 },
-    { 0x6C, 0x0 },
-    { 0x6D, 0x0 },
-    { 0x6E, 0x0 },
-    { 0x2F, 0x0 },
-    { 0x31, 0x0 },
-    { 0x46, 0x0 }
+    { 0x5A, 0x1 }, // Jubilife TV
+    { 0x5B, 0x0 }, // Jubilife Poketch
+    { 0x5C, 0x1 }, // ???
+    { 0x5D, 0x1 }, // Jubilife Trainer School
+    { 0x5E, 0x0 }, // Oreburgh Museum
+    { 0x5F, 0x1 }, // Floaroma Flower Shop
+    { 0x60, 0x1 }, // Eterna Flower Shop
+    { 0x61, 0x1 }, // Hearthome Contest Hall
+    { 0x62, 0x1 }, // Hearthome Poffin House
+    { 0x63, 0x0 }, // Hearthome Foreign Building
+    { 0x64, 0x1 }, // Solaceon Day Care
+    { 0x65, 0x0 }, // Veilstone Department Store
+    { 0x66, 0x1 }, // Veilstone Game Corner
+    { 0x47, 0x0 }, // Galactic HQ
+    { 0x67, 0x0 }, // Canalave Library
+    { 0x40, 0x0 }, // Snowpoint Temple
+    { 0x68, 0x0 }, // Sunyshore Lighthouse
+    { 0x69, 0x1 }, // Sunyshore Market
+    { 0x50, 0x1 }, // Battle Tower
+    { 0x6A, 0x0 }, // Pokemon Mansion
+    { 0x6B, 0x1 }, // Footstep House
+    { 0x6C, 0x0 }, // ??? Cafe
+    { 0x6D, 0x0 }, // Grand Lake
+    { 0x6E, 0x0 }, // ??? Restaurant
+    { 0x2F, 0x0 }, // Valley Windworks
+    { 0x31, 0x0 }, // Fuego Ironworks
+    { 0x46, 0x0 } // Old Chateau
```

Comment:
They're not actually location names, they're `mapLabelTextID`s. Are there constants for that as well? I couldn't find them, which is why I did it like this for now until those do exist, but I understand not wanting to have comments like this.

Reply:
The constants to use are precisely those from `message_bank_location_names.gmm`, and `mapLabelTextID` is actually a misnomer. I think it's ok to hold off using the constants for now, and rename the field + use constants in another PR (the comments can be removed for now though).

Reply:
Oh wait I misread what Lhea said, mixed it up with map header constants. I'll just add the constants from `message_bank_location_names.gmm` if I'm removing the comments anyways.

Reply:
That's fine, but next time, if you want to rename the constants (as you've already done in this PR), please do it another PR instead so that each PR can remain focused.